src/seedu/addressbook/Main.java

```diff
@@ -111,10 +111,13 @@ private CommandResult executeCommand(Command command)  {
             CommandResult result = command.execute();
             storage.save(addressBook);
             return result;
+        } catch (StorageOperationException e) {
+            ui.showToUser(e.getMessage());
```

Comment:
As discussed in the tutorial, are there any better ways to handle this exception instead of just telling the user what exception occurred?